templates/base/head.tmpl

```diff
@@ -3,8 +3,19 @@
 <head data-suburl="{{AppSubUrl}}">
 	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
 	<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-	<meta name="author" content="{{if .Repository }}{{.Owner.Name}}{{else}}Gogs - Go Git Service{{end}}" />
-	<meta name="description" content="{{if .Repository }}{{.Repository.Name}} - {{.Repository.Description}}{{else}}Gogs (Go Git Service) is a painless self-hosted Git service written in Go{{end}}" />
+	<meta name="author" content="
+	{{- if .Repository -}}
```

Comment:
Please do not use template syntax only available in go1.6, Gogs has backward compatibility to go1.4.